content/en/real_user_monitoring/mobile_and_tv_monitoring/flutter/troubleshooting.md

```diff
@@ -62,6 +62,19 @@ LoadError - dlsym(0x7fbbeb6837d0, Init_ffi_c): symbol not found - /Library/Ruby/
 
 Follow the instructions in the [Flutter documentation][2] for working with Flutter on Apple Silicon.
 
+## Undefined symbol (iOS)
+
+If you are using Flutter's `build ios-framework` command, it is possible you will get errors similar to the following:
+
+```
+Undefined symbol: _OBJC_CLASS_$_PLCrashReport
+Undefined symbol: _OBJC_CLASS_$_PLCrashReportBinaryImageInfo
+Undefined symbol: _OBJC_CLASS_$_PLCrashReportStackFrameInfo
+...
+```
+
+This is because the `build ios-framework` command is not properly including PLCrashReporter, which the Datadog Flutter SDK depends on. To solve this issue, we recommend you include the PLCrashReporter dependency manually. The framework and instructions for including it are available on its [GitHub page][8]
```

Comment:
```suggestion
This occurs because the `build ios-framework` command does not properly include PLCrashReporter, which the Datadog Flutter SDK depends on. To resolve this issue, Datadog recommends manually including the PLCrashReporter dependency. The framework and instructions for including it are available on its [GitHub page][8].
```